signing_service/signing_service/signing_service.py

```diff
@@ -384,6 +419,33 @@ def _get_authentication_challenge_signature(self, authentication_challenge: byte
             authentication_challenge,
         )
 
+    @staticmethod
+    def _get_daily_transaction_threshold_file_path() -> PosixPath:
+        thresholds_directory = (Path.cwd()).joinpath('daily_tresholds')  # pylint: disable=no-member
+        thresholds_directory.mkdir(exist_ok=True)  # pylint: disable=no-member
+        daily_file = datetime.datetime.now().strftime('%Y-%m-%d')
+        thresholds_directory.joinpath(daily_file).touch(exist_ok=True)  # pylint: disable=no-member
+        thresholds_directory = thresholds_directory.joinpath(daily_file)
+        return thresholds_directory
+
+    def _get_signing_service_daily_transaction_sum_so_far(self) -> int:
+        threshold_file = self._get_daily_transaction_threshold_file_path()
+        try:
+            transaction_sum = int(threshold_file.read_text())  # pylint: disable=no-member
+        except ValueError:
+            threshold_file.write_text('0')  # pylint: disable=no-member
+            return 0
+        return transaction_sum
+
+    def _add_payload_value_to_daily_transactions_sum(self, transactions_sum: int) -> None:
+        thresholds_directory = self._get_daily_transaction_threshold_file_path()
+        thresholds_directory.write_text(str(transactions_sum))  # pylint: disable=no-member
+
+    def _update_last_signed_transaction_date(self, last_transaction_date: str) -> None:
```

Comment:
This function name describes something else that it is actually doing. The function is not updating the date but loading the value when date changes? Maybe apart from fixing the name there should be some comment here.

Reply:
Done